Sources/SecureXPC/Server/ClientRequirement.swift

```diff
@@ -60,6 +60,13 @@ extension XPCServer.ClientRequirement {
             return try teamIdentifier(teamID)
         }
     }
+
+	/// The requesting client must have hardened runtime enabled.
```

Comment:
From looking at [Apple's documentation](https://developer.apple.com/documentation/security/hardened_runtime) it appears they always refer to this as a proper noun capitalized as "Hardened Runtime".